webpack.config.js

```diff
@@ -1,31 +1,50 @@
-var webpack = require('webpack');
+var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 module.exports = {
+  mode: 'production',
   entry: './src/index',
-  module: {
-    loaders: [
-      { test: /\.json$/, loader: 'json' },
-      { test: /\.js$/, loader: 'babel', exclude: /node_modules/ }
-    ]
-  },
   output: {
-    filename: 'dist/markdown-react.min.js',
+    filename: 'markdown-react.min.js',
```

Comment:
why removing dist?

Reply:
I deleted the `dist` folder and re-ran `npm i` as well as `npm run dist`. It generated the `dist` folder automatically.